libs/content/workspace/src/workspace.rs

```diff
@@ -395,6 +395,13 @@ impl Workspace {
                                 if resp.hide_virtual_keyboard {
                                     output.hide_virtual_keyboard = resp.hide_virtual_keyboard;
                                 }
+                                if resp.text_updated {
+                                    output.markdown_editor_text_updated = true;
+                                }
+                                if resp.selection_updated || resp.scroll_updated {
+                                    // not sure what scroll has to do with it but I'm just moving code written by smail
```

Comment:
`markdown_editor_selection_updated` signifies whether or not cursor physically moved around on screen. I use the field to signal that the native cursor's bounds (on the swift side) should be recomputed. That can happen when the selection is updated, or when they scroll (cursor changes position on screens).